README-Python-Dev.md

```diff
@@ -0,0 +1,76 @@
+# Developing a Python dependency of this project
+
+This document contains a suggested approach on how to develop or modify a Python library that uses the current project Docker as a test or develop environment.
+For example, those instructions can be used to create a developer environment for further development of [eudi-wallet-it-python](https://github.com/italia/eudi-wallet-it-python).
+The instructions below are intended to be a suggestion or a guideline rather than a standard.
+
+## Step 0: Identify which Python dependency requires development
+
+We will assume that the developer want to develop a modified version of the library [eudi-wallet-it-python](https://github.com/italia/eudi-wallet-it-python) which is a dependency of the container `satosa-saml2spid`.
```

Comment:
```suggestion
We will assume that the developer needs to develop a modified version of the library [eudi-wallet-it-python](https://github.com/italia/eudi-wallet-it-python) which is a dependency of the container `satosa-saml2spid`.
```

Reply:
applied in af64492ef64e46ab1a5cdf11bbece22536b38317